src/main/java/gov/nasa/jpl/mbee/mdk/generator/DocumentGenerator.java

```diff
@@ -277,11 +278,20 @@ public Section parseView(Element view) {
                     //if (expose.size() == 1 && expose.get(0) instanceof Diagram) {
                     for (Element ex : elementImports) {
                         if (ex instanceof Diagram) {
-                            Image image = new Image();
-                            List<Object> images = new ArrayList<Object>();
-                            images.add(ex);
-                            image.setTargets(images);
-                            viewSection.addElement(image);
+                            DiagramType diagramType = Application.getInstance().getProject().getDiagram((Diagram) ex).getDiagramType();
+                            if(diagramType.isTypeOf(DiagramType.GENERIC_TABLE)||diagramType.getType().equals(MatricesAndTables.INSTANCE_TABLE) || diagramType.getType().equals(MatricesAndTables.VERIFY_REQUIREMENTS_MATRIX) || diagramType.getType().equals(MatricesAndTables.ALLOCATION_MATRIX) || diagramType.getType().equals(MatricesAndTables.SATISFY_REQUIREMENTS_MATRIX)){
```

Comment:
I don't know what a dependency matrix is but none of the above is a subtype of it.
My MD doesn't offer me the creation of a dependency matrix.